azure/iac/master-compliance-test.json

```diff
@@ -3587,6 +3587,111 @@
                         "microsoft.cache/redis"
                     ]
                 },
+                {
+                    "masterTestId": "TEST_Redis_4",
```

Comment:
@shahinshirvani change this to `"masterTestId": "TEST_Redis_3",` and so on for the rest of the items.